packages/frontend/src/components/SimpleTable/index.tsx

```diff
@@ -139,6 +141,13 @@ const SimpleTable: FC<SimpleTableProps> = ({
                                 itemsMap={itemsMap}
                             />
                         );
+                    const cell = props.cell;
+                    if (
+                        cell.getValue() === undefined ||
```

Comment:
That's a really nice catch. It's especially good because the changes in this file are no longer needed, since the changes to TableBody.tsx were recently added. Removes redundant code.

Reply:
perfect!